src/backend/utils.ts

```diff
@@ -278,6 +279,7 @@ export const getSystemInfo = async () => {
 
   return `Heroic Version: ${heroicVersion}
 Legendary Version: ${legendaryVersion}
+GogDL Version: ${gogdlVersion}
```

Comment:
Should this be `GOGdl`? cc @imLinguin to confirm, I think I remember some comment about this on discord

Reply:
fixed